bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/RuleEngineImpl.java

```diff
@@ -1032,6 +1032,15 @@ public void runNow(String ruleUID, boolean considerConditions, @Nullable Map<Str
             } else {
                 executeActions(rule, false);
             }
+            if (context != null) {
```

Comment:
I don't love having to catch an exception here for `runNow` to return outputs, but I was wary of changing the return type to `Map<String, Object>` in order to just return the new context directly, since it's technically an interface change.